dkms.8.in

```diff
@@ -572,12 +641,16 @@ if MAKE[0] should be used to build the module for that kernel. If multiple
 .B MAKE_MATCH
 directives match against the kernel being built for, the last matching
 .B MAKE[#]
-will be used to build your module. If no MAKE directive is specified or if no
-MAKE_MATCH matches the kernel being built for, DKMS
-will attempt to use a generic MAKE command to build your module.
+will be used to build your module. If no
+.B MAKE
+directive is specified or if no
+.B MAKE_MATCH
+matches the kernel being built for, DKMS will attempt to use a generic make command to build your module.
 
-KERNELRELEASE will be automatically appended to MAKE[#]. If you want to
-suppress this behavior, you can quote the make command: 'make'.
+.B $KERNELRELEASE
```

Comment:
actually `KERNELRELEASE=$kernelver` gets injected, along with `-j$parallel_jobs`

Reply:
Yep, forgot to add, had the same thought while going through the man page. Added, thanks.